src/simtools/simtel/simulator_array.py

```diff
@@ -86,7 +87,7 @@ def _make_run_command(self, run_number=None, input_file=None):
         command += super().get_config_option("output_file", output_file)
         command += super().get_config_option("random_state", "none")
         if self.sim_telarray_seeds and self.sim_telarray_seeds.get("random_instrument_instances"):
-            config_dir = Path(self.corsika_config.array_model.config_file_path).parent
+            config_dir = Path(config_file_path).parent
```

Comment:
See comment above.

Reply:
This is the directory - so I've changed it.